appeals/web/src/server/appeals/appeal-details/costs/costs.mapper.js

```diff
@@ -0,0 +1,92 @@
+import config from '#environment/config.js';
+import { appealShortReference } from '#lib/appeals-formatter.js';
+
+/**
+ * @typedef {Object} CostsDocumentType
+ * @property {number} id
+ * @property {string} name
+ */
+
+/**
+ *
+ * @param {import('../appeal-details.types.js').WebAppeal} appealDetails
+ * @param {CostsDocumentType[]} documentTypes
+ * @returns {PageContent}
+ */
+export function addDocumentTypePage(appealDetails, documentTypes) {
+	const shortAppealReference = appealShortReference(appealDetails.appealReference);
+
+	/** @type {PageContent} */
+	const pageContent = {
+		title: `What is the document type? - ${shortAppealReference}`,
+		backLinkUrl: `/appeals-service/appeal-details/${appealDetails.appealId}`,
+		heading: 'What is the document type?',
```

Comment:
Are you not missing the preheading with "Appeal ${short appeal reference}?

Reply:
It's not displayed on the page in either the mural or the prototype so I think it's been intentionally omitted by design